engine/Shopware/Components/Plugin.php

```diff
@@ -284,4 +287,41 @@ private function registerFilesystem(ContainerBuilder $container, $key)
 
         $container->setDefinition($serviceId, $filesystem);
     }
+
+    private function registerLogger(ContainerBuilder $container)
+    {
+        $logLevelParameterName = sprintf('%s.logger.level', $this->getContainerPrefix());
+        if (!$container->hasParameter($logLevelParameterName)) {
+            $container->setParameter($logLevelParameterName, Logger::DEBUG);
```

Comment:
```suggestion
            $container->setParameter($container->getParameter('kernel.default_error_level'));
```
You don't want it to Debug

Reply:
Good hint. Done

Reply:
uhm this parameter seems not to be available as seen in the [error in the job](https://travis-ci.org/shopware/shopware/jobs/502436528#L1378).

Reply:
Hmmmm https://github.com/shopware/shopware/blob/5.5/engine/Shopware/Kernel.php#L747

Reply:
So I rather use `shopware.logger.level`? This should be the same